crates/re_data_ui/src/image.rs

```diff
@@ -230,26 +232,77 @@ fn texture_size(colormapped_texture: &ColormappedTexture) -> Vec2 {
     egui::vec2(w as f32, h as f32)
 }
 
-fn show_image_at_max_size(
+#[derive(Clone, Copy, PartialEq, Eq)]
+enum PreviewSizeShrink {
+    /// Stick to the desired size. May cause
```

Comment:
…irritated bowls?

Reply:
May cause copy and paste error without reading over it again. :nod: